### PR TITLE
fix(ls): actually send ls metrics

### PR DESCRIPTION
**Pull request by ajbt200128**
we weren't sending metrics for the LS, now we are. See this interfaces PR for what I've added to metrics: https://github.com/semgrep/semgrep-interfaces/pull/218

## Test plan
I have no test plan 🤷

### CODE REVIEW

**Review by aryx — APPROVED** [review]
Hehe :) now we know LSP is making some web requests! Thx to capabilities!

BTW last time when I made http_helpers use capabilities, I've added a few
networks_caps_UNSAFE in Session.ml. Maybe we can remove them! Ideally we should not have any of those xxx_UNSAFE.

File: cli/src/semgrep/metrics.py [hunk 1]
```diff
@@ -135,6 +135,8 @@ class Metrics:
                 projectHash=None,
                 ci=None,
                 isDiffScan=False,
+                os=os.name,
```
Comment (zzeleznick) [inline cli/src/semgrep/metrics.py]: can we use `platform.system()` 

see https://docs.python.org/3/library/platform.html

File: src/osemgrep/core/Metrics_.ml [hunk 2]
```diff
@@ -136,6 +136,8 @@ let default_payload =
     environment =
       {
         version = Version.version;
+        os = Sys.os_type;
```
Comment (zzeleznick) [inline src/osemgrep/core/Metrics_.ml:139]: oh no :( 

looks like ocaml doesn't seem to distinguish mac vs linux out-of-the-box.

see https://github.com/ocaml/opam/issues/3058 

(also hannes is on the thread 😆)

File: cli/src/semgrep/metrics.py (hunk 1 above)
Comment (ajbt200128) [reply cli/src/semgrep/metrics.py]: Java is a system 💀 but yea this makes sense

File: src/osemgrep/core/Metrics_.ml (hunk 2 above)
Comment (ajbt200128) [reply src/osemgrep/core/Metrics_.ml:139]: ugh annoying, we'll just have to fix it in metabase or smthn

**Review by zzeleznick — APPROVED** [review]

**Pull request merged**